test/_apm_server.js

```diff
@@ -78,6 +75,10 @@ function APMServer (agentOpts, mockOpts = { expect: [] }) {
 
         index++
       })
+      parsedStream.on('end', function () {
```

Comment:
Question: Why end with a 202? Is this just mimicking APM server's HTTP response code, or is there more going on here?

Reply:
Yes, just mimicking APM server's HTTP response code.

Some background: In earlier work of the `makeInjestRequest()` re-write in the http client I had initially only accepted a "202" response from APM server. However, I ended up relaxing that to accepting any "2xx" response because we have test suites using mock APM servers (like this one) that just `res.end()` defaulting to a "200" response.